### PR TITLE
fix link in syntax.md

### PR DESCRIPTION
**Pull request by zyctree**

### CODE REVIEW

**Comment by Veykril** [comment]
Thanks!
@bors r+

File: docs/dev/syntax.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The things described are implemented in three places
 * [rowan](https://github.com/rust-analyzer/rowan/tree/v0.9.0) -- a generic library for rowan syntax trees.
 * [ra_syntax](https://github.com/rust-lang/rust-analyzer/tree/cf5bdf464cad7ceb9a67e07985a3f4d3799ec0b6/crates/ra_syntax) crate inside rust-analyzer which wraps `rowan` into rust-analyzer specific API.
   Nothing in rust-analyzer except this crate knows about `rowan`.
-* [parser](https://github.com/rust-lang/rust-analyzer/tree/cf5bdf464cad7ceb9a67e07985a3f4d3799ec0b6/crates/parser) crate parses input tokens into an `ra_syntax` tree
+* [parser](https://github.com/rust-lang/rust-analyzer/tree/cf5bdf464cad7ceb9a67e07985a3f4d3799ec0b6/crates/ra_parser) crate parses input tokens into an `ra_syntax` tree
```
Comment (lnicola) [inline docs/dev/syntax.md:14]:
```suggestion
* [ra_parser](https://github.com/rust-lang/rust-analyzer/tree/cf5bdf464cad7ceb9a67e07985a3f4d3799ec0b6/crates/ra_parser) crate parses input tokens into an `ra_syntax` tree
```

**Comment by lnicola** [comment]
It might also be nice to point these to a more recent commit.

**Comment by Veykril** [comment]
Oh wait, the patch is backwards since the crates lost the `ra_` prefix 😅 Shouldn't be reviewing first thing in the morning

**Comment by lnicola** [comment]
Well, the hash is still pointing to the older crate names, but yeah, it should be updated.

**Comment by lnicola** [comment]
@zyctree what we're trying to say is that since the docs were written, the crates were renamed to drop the `ra_` prefix, as you can see in https://github.com/rust-lang/rust-analyzer/tree/36a70b7/crates. Can you update the docs to point to a newer commit and the correct crate names?

**Comment by zyctree** [comment]
@lnicola I updated, using the commit you just mentioned.
And the beginning of docs says it is the status of 2020-01-09, maybe this date should be updated as well.

File: docs/dev/syntax.md [hunk 2]
```diff
@@ -9,9 +9,9 @@ This guide describes the current state of syntax trees and parsing in rust-analy
 The things described are implemented in three places
 
 * [rowan](https://github.com/rust-analyzer/rowan/tree/v0.9.0) -- a generic library for rowan syntax trees.
```
Comment (lnicola) [inline docs/dev/syntax.md:11]: Maybe point this to `v0.15.10`?

File: docs/dev/syntax.md [hunk 3]
```diff
@@ -9,9 +9,9 @@ This guide describes the current state of syntax trees and parsing in rust-analy
...
-* [ra_syntax](https://github.com/rust-lang/rust-analyzer/tree/cf5bdf464cad7ceb9a67e07985a3f4d3799ec0b6/crates/ra_syntax) crate inside rust-analyzer which wraps `rowan` into rust-analyzer specific API.
+* [syntax](https://github.com/rust-lang/rust-analyzer/tree/36a70b7435c48837018c71576d7bb4e8f763f501/crates/syntax) crate inside rust-analyzer which wraps `rowan` into rust-analyzer specific API.
   Nothing in rust-analyzer except this crate knows about `rowan`.
-* [parser](https://github.com/rust-lang/rust-analyzer/tree/cf5bdf464cad7ceb9a67e07985a3f4d3799ec0b6/crates/parser) crate parses input tokens into an `ra_syntax` tree
+* [parser](https://github.com/rust-lang/rust-analyzer/tree/36a70b7435c48837018c71576d7bb4e8f763f501/crates/parser) crate parses input tokens into an `syntax` tree
```
Comment (lnicola) [inline docs/dev/syntax.md:14]:
```suggestion
* [parser](https://github.com/rust-lang/rust-analyzer/tree/36a70b7435c48837018c71576d7bb4e8f763f501/crates/parser) crate parses input tokens into a `syntax` tree
```

**Review by lnicola — APPROVED** [review]

**Pull request merged**